plugins/register-component/src/components/RegisterComponentForm/RegisterComponentForm.tsx

```diff
@@ -71,7 +71,7 @@ const RegisterComponentForm: FC<Props> = ({ onSubmit, submitting }) => {
           name="componentLocation"
           required
           margin="normal"
-          helperText="Enter the full path to the component.yaml file in GitHub to start tracking your component. It must be in a public repo."
+          helperText="Enter the full path to the component.yaml file in Github, Gitlab, Bitbucket or Azure to start tracking your component. For private repo provide authentication information via config."
```

Comment:
Please use proper CamelCasing of GitHub and GitLab. I just changed all of them 😎

Reply:
@stefanalund thanks for the review. updated the casing. is there anything else you want me to change in this PR

Reply:
@o-farooq great! I'll leave it to @soapraj to do a full review since the rest is out of my comfort zone 😬